cli/tests/fixtures/metronome.py

```diff
@@ -107,14 +107,14 @@ def job_history_fixture():
 
     return [
       {
-        "createdAt": "2017-03-31T21:05:32.422+0000",
-        "finishedAt": "2017-03-31T21:05:46.805+0000",
-        "id": "20170331210532uxgVF"
+        "createdAt": "2017-09-21T21:20:22.581+0000",
+        "finishedAt": "2017-09-21T21:20:25.315+0000",
+        "id": "20170921212022CrNjS"
       },
       {
-        "createdAt": "2017-03-31T21:05:30.613+0000",
-        "finishedAt": "2017-03-31T21:05:41.740+0000",
-        "id": "20170331210530QHpRU"
```

Comment:
If these tests were failing we should improve them not to be time-dependent, otherwise please revert, this introduces unnecessary changes to the review.

Reply:
yeah... tests and changes were frustrating... always have been.   the alignment of the text is checked, however there are either extra characters or linefeeds on mac that do not allow for easy creation of the test files... I'll take a look again.

Reply:
I reverted to reduce the change... however there are 2 changes that require changes to the test and results:
1. ID -> TASK ID
2. the order is sorted by time 
none of which requires a change to the fixture.